dist/stylesheet.css

```diff
@@ -250,3 +250,53 @@
   margin-right: 60px;
   margin-top: 0;
 }
+
+.aw_button_clicked_form {
+  background: grey;
+}
+
+.aw_review_modal {
+  position: absolute;
```

Comment:
If you want it to remain directly in front of the user even if they scroll you can use `position: fixed;`